docs/community.md

```diff
@@ -0,0 +1,14 @@
+# Community
+
+We welcome your feedback, questions and ideas. Here's how to reach the community.
+
+## Forum
+
+Join the [Ansible Forum](https://forum.ansible.com) as a single starting point and our default communication platform for questions and help, development discussions, events, and much more. [Register](https://forum.ansible.com/signup?) to join the community. Search by categories and tags to find interesting topics or start a new one; subscribe only to topics you need!
+
+* [Get Help](https://forum.ansible.com/c/help/6): get help or help others. Please add appropriate tags if you start new discussions, for example `antsibull`.
```

Comment:
I don't think Get Help makes sense here. Collection Development is way better suited as users of antsibull-changelog will almost always be collection developers.

Reply:
we should rename it to "Give Help" here:) anyway, SGTM